python/rikai/types/geometry.py

```diff
@@ -281,6 +281,36 @@ def _area(arr: np.ndarray) -> np.array:
             0, arr[:, 3] - arr[:, 1]
         )
 
+    @staticmethod
+    def iou_matrix(box_list1: Union[Sequence[Box2d], np.ndarray],
+                   box_list2: Union[Sequence[Box2d], np.ndarray]) -> np.ndarray:
+        """Compute intersection over union(IOU).
```

Comment:
We use [numpy doc style](https://numpydoc.readthedocs.io/en/latest/format.html) in rikai. could you help to update the document to be aligned with the docstring style?